app/builders/messages/instagram/message_builder.rb

```diff
@@ -0,0 +1,149 @@
+# This class creates both outgoing messages from chatwoot and echo outgoing messages based on the flag `outgoing_echo`
+# Assumptions
+# 1. Incase of an outgoing message which is echo, source_id will NOT be nil,
+#    based on this we are showing "not sent from chatwoot" message in frontend
+#    Hence there is no need to set user_id in message for outgoing echo messages.
+
+class Messages::Instagram::MessageBuilder < Messages::Messenger::MessageBuilder
+  attr_reader :messaging
+
+  def initialize(messaging, inbox, outgoing_echo: false)
+    super()
```

Comment:
do we need the super?

Reply:
No, but rubocop gives an error. :(

Reply:
ah makes sense :P no harm anyways. so let's go ahead with it.